tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/SModelCreatorHelper.java

```diff
@@ -0,0 +1,34 @@
+package org.palladiosimulator.simexp.dsl.smodel.interpreter.util;
```

Comment:
Should this class not rather reside in the test fragment?

Reply:
Isn't org.palladiosimulator.simexp.dsl.smodel.interpreter.tests the test fragment?